src/utilities/network/lookupApi.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2020
+ */
+
+import { SETTINGS_KEYS } from '../../settings';
+import { UIDatabase } from '../../database';
+
+const { SYNC_URL } = SETTINGS_KEYS;
+
+const RESOURCES = {
+  PATIENT: '/api/v4/patient',
+  PRESCRIBER: '/api/v4/prescriber',
+};
+
+const SEPARATORS = {
+  QUERY_STRING: '?',
+  QUERY_PARAMETERS: '&',
+  POLICY_NUMBER: '-',
+};
+
+const getQueryString = ({
+  firstName = '',
+  lastName = '',
+  registrationCode = '',
+  dateOfBirth = '',
+  policyNumberPerson = '',
+  policyNumberFamily = '',
+} = {}) => {
+  const policyNumberSeparator =
+    policyNumberPerson && policyNumberFamily ? SEPARATORS.POLICY_NUMBER : '';
+  const policyNumber = policyNumberPerson + policyNumberSeparator + policyNumberFamily;
```

Comment:
wrt the mSupply issue [#5680](https://github.com/sussol/msupply/pull/5860), there is a possibility that the `policyNumberPerson` would be blank in which case there shouldn't be a separator. ( another instance where the C# `Trim([char])` is great, but not available in js )

Reply:
Good point. I think we've almost got agreement on the insurance policy number spec, but will wait until it's implemented just to be safe. Have updated epic with issue.